pandas/tests/frame/test_arithmetic.py

```diff
@@ -252,6 +253,48 @@ def test_arith_flex_zero_len_raises(self):
 
 
 class TestFrameArithmetic(object):
+    # TODO: tests for other arithmetic ops
+    def test_df_add_2d_array_rowlike_broadcasts(self):
```

Comment:
can you expand to use the all_arithmetic_ops fixture? (or some of them)?

Reply:
Sure.  I'll keep these tests as-is because they have nice explicitly-written-out `expected`s, and add another pair of tests using the fixtures.